tests/storage/volume_metadata_test.py

```diff
@@ -71,6 +71,7 @@ def make_md_dict(**kwargs):
         sc.LEGALITY: 'legality',
         sc.CTIME: '0',
         sc.GENERATION: '1',
+        sc.SEQUENCE: sc.DEFAULT_SEQUENCE,
```

Comment:
I think we should use non default value here and in make_init_params:

    sc.SEQUENCE: 7,

This and the other changes in this module should really be done in the previous commit.
This commit should only modify the api layers and tests affected by the new API
(blocksd_test,localfssd_test,sdm_redirection_test). This module should be complete
in the previous commit.

Reply:
good idea, `test_sequence_default` was actually incorrect and it passed